content/sensu-enterprise/3.1/quick-start/five-minute-install.md

```diff
@@ -0,0 +1,283 @@
+name=rabbitmq-erlang
+baseurl=https://dl.bintray.com/rabbitmq/rpm/erlang/20/el/7
+gpgcheck=1
+gpgkey=https://www.rabbitmq.com/rabbitmq-release-signing-key.asc
+repo_gpgcheck=0
+enabled=1' | sudo tee /etc/yum.repos.d/rabbitmq-erlang.repo{{< /highlight >}}
+
+Install Erlang (required for RabbitMQ):
+
+{{< highlight shell >}}
+sudo yum install erlang -y{{< /highlight >}}
+
+Install RabbitMQ:
+
+{{< highlight shell >}}
+sudo yum install https://dl.bintray.com/rabbitmq/rabbitmq-server-rpm/rabbitmq-server-3.6.12-1.el7.noarch.rpm -y{{< /highlight >}}
+
+Configure RabbitMQ to work with Sensu:
+
+{{< highlight shell >}}
+echo '{
+  "rabbitmq": {
+    "host": "127.0.0.1",
+    "port": 5672,
+    "vhost": "/sensu",
+    "user": "sensu",
+    "password": "secret",
+    "heartbeat": 30,
+    "prefetch": 50
+  }
+}' | sudo tee /etc/sensu/conf.d/rabbitmq.json{{< /highlight >}}
+
+Start RabbitMQ:
+
+{{< highlight shell >}}
+sudo systemctl start rabbitmq-server
+sudo systemctl enable rabbitmq-server{{< /highlight >}}
+
+**7. Install Sensu Enterprise:**
+
+{{< highlight shell >}}
+sudo yum install sensu sensu-enterprise sensu-enterprise-dashboard -y{{< /highlight >}}
+
+**8. Configure Sensu Enterprise server:**
+
+Run the following to set up a minimal server config:
+
+{{< highlight shell >}}
+echo '{
+  "transport": {
+    "name": "rabbitmq"
+  },
+  "api": {
+    "host": "127.0.0.1",
+    "port": 4567
+  }
+}' | sudo tee /etc/sensu/config.json{{< /highlight >}}
+
+**9. Configure the Sensu client:**
+
+Run the following to set up a minimal client config:
+
+{{< highlight shell >}}
+echo '{
+  "client": {
+    "environment": "development",
+    "subscriptions": [
+      "dev"
+    ]
+  }
+}' |sudo tee /etc/sensu/conf.d/client.json{{< /highlight >}}
+
+**10. Configure the Sensu Enterprise dashboard:**
+
+{{< highlight shell >}}
+echo '{
+  "sensu": [
+    {
+      "name": "sensu",
+      "host": "127.0.0.1",
+      "port": 4567
+    }
+  ],
+  "dashboard": {
+    "host": "0.0.0.0",
+    "port": 3000
+  }
+}' |sudo tee /etc/sensu/dashboard.json{{< /highlight >}}
+
+**11. Make sure that the `sensu` user owns all of the Sensu configuration files:**
+
+{{< highlight shell >}}
+sudo chown -R sensu:sensu /etc/sensu{{< /highlight >}}
+
+**12. Start the Sensu services:**
+
+{{< highlight shell >}}
+sudo systemctl enable sensu-{enterprise,enterprise-dashboard,client}
+sudo systemctl start sensu-{enterprise,enterprise-dashboard,client}
+{{< /highlight >}}
+
+Nice work! You have successfully installed and configured Sensu.
+
+You can verify that your installation is ready to use by querying the Sensu API
+using the `curl` utility (and piping the result to the [`jq` utility][10]):
+
+{{< highlight shell >}}
+sudo yum install jq curl -y
+curl -s http://127.0.0.1:4567/clients | jq .{{< /highlight >}}
+
+The Sensu Enterprise API should return a JSON array of Sensu clients similar to this:
+
+{{< highlight shell >}}
+$ curl -s http://127.0.0.1:4567/clients | jq .
```

Comment:
This command is repeated from line 231. Maybe remove one? Not sure which makes the most sense.